recipes-connectivity/wpa-supplicant/wpa-supplicant_2.%.bbappend

```diff
@@ -5,11 +5,18 @@ SRC_URI =+ "file://wep-hexkeys-and-wpa-psk-hexkeys-fix.patch \
             file://dbus-wpa_supplicant.conf;subdir=nilrt \
             file://fi.epitest.hostap.WPASupplicant.service;subdir=nilrt \
             file://fi.w1.wpa_supplicant1.service;subdir=nilrt \
+            file://wpa_supplicant_conf;subdir=nilrt \
 "
 
 PACKAGECONFIG_append = " openssl"
 
 do_install_append () {
 	install -m 644 ${WORKDIR}/nilrt/dbus-wpa_supplicant.conf ${D}/${sysconfdir}/dbus-1/system.d
 	install -m 644 ${WORKDIR}/nilrt/*.service ${D}/${datadir}/dbus-1/system-services
+
+	# Remove the wpa_supplicant.conf file installed by the OE recipe. This will be replaced by
```

Comment:
If we rename wpa_supplicant_conf to wpa_supplicant.conf, wouldn't bitbake automatically replace OE's version with this?

Reply:
nvm. I thought you were installing the file as wpa_supplicant.conf.

Reply:
wpa_supplicant_conf is a shell script. Lines 7-12 in that script generate the .conf file if it doesn't already exist. So it is not a 1-1 replacement.

Reply:
Yes, hadn't read the description properly